src/ProjectTemplates/test/Templates.Blazor.Tests/BlazorWasmTemplateTest.cs

```diff
@@ -107,7 +107,7 @@ private static async Task AssertCompressionFormat(AspNetProcess aspNetProcess, s
         Assert.Equal(expectedEncoding, response.Content.Headers.ContentEncoding.Single());
     }
 
-    [Theory(Skip = "https://github.com/dotnet/aspnetcore/issues/30882")]
+    [Theory(Skip = "https://github.com/dotnet/aspnetcore/issues/45736")]
```

Comment:
The issue here was blocked on a Playwright issue, which they have resolved with the following comment: 
<img width="690" alt="image" src="https://user-images.githubusercontent.com/34246760/216194844-361796de-0ed5-42ab-81e4-67e4f2d2fa66.png">

Does that impact our decisions regarding this in any way?
I assume you're trying to wrap up your work already, and leaving this to be handled as part of the work on the linked issue is the right thing to do. So mostly wanted to note the progress on the dependency.

Reply:
Going to try un-skipping this

Reply:
Nope, still failing